core/src/main/java/com/sequenceiq/cloudbreak/service/stack/StackFailureHandlerService.java

```diff
@@ -108,22 +108,29 @@ private void handleExceptions(Stack stack, List<ResourceRequestResult> failedRes
                     resourceList.add(newResource);
                 }
             }
+            for (Resource resource : exception.getBuiltResources()) {
```

Comment:
We can exclude this to a method as it's the same as above they just act on a different resource list but the logic is the same